components/automate-prometheus/habitat/config/prometheus.yml

```diff
@@ -2,7 +2,7 @@
 #
 # my global config
 global:
-  scrape_interval:     1m # Set the scrape interval to every 15 seconds. Default is every 1 minute.
+  scrape_interval:     15s # Set the scrape interval to every 15 seconds. Default is every 1 minute.
```

Comment:
prometheus causes very little load as far as I can tell. It's nicer to have the data refresh faster when you start testing something.